src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -881,27 +872,26 @@ export function getNextEvents(currentStatus, details) {
       const remandSsocDuration = makeDurationText(details.remandSsocTimeliness);
       const formattedSocDate = moment(details.lastSocDate, 'YYYY-MM-DD').format('MMMM Do, YYYY');
       return {
-        header: 'What happens next depends on whether you send in new evidence.',
+        header: 'What happens next depends on whether you submit new evidence.',
         events: [
           {
             title: 'Your appeal will be returned to the Board',
             description: (
               <p>
-                <strong>If you don’t send in new evidence after the Statement of the Case
-                on {formattedSocDate}</strong>, the Veterans Benefits Administration will
-                finish their work on the remand and return your case to the Board of
-                Veterans’ Appeals.
+                <strong>If you don’t submit new evidence after the Supplemental Statement of the Case on {formattedSocDate}</strong>, the Veterans Benefits Administration will
+                finish their work on the remand and return your case to the Board of Veterans’ Appeals.
               </p>
             ),
             durationText: returnSsocDuration.header,
             cardDescription: `The Veterans Benefits Administration typically takes ${returnSsocDuration.description} to return cases to the Board.`,
           }, {
             title: 'The Veterans Benefits Administration will send you a new Statement of the Case',
+            // TODO: should "new" be removed from before SSoC, it is removed from previous case.
             description: (
               <p>
-                <strong>If you send in new evidence after the Statement of the Case
+                <strong>If you submit new evidence after the Supplemental Statement of the Case
                 on {formattedSocDate}</strong>, the Veterans Benefits Administration
-                will need to write a new Statement of the Case before returning your
+                will need to write a new Supplemental Statement of the Case before returning your
```

Comment:
@gnakm I believe this is what @cehsu was asking about [here](https://github.com/department-of-veterans-affairs/vets-website/pull/7374/files#r172103226). It parallels my question above: Should this be "new Supplemental Statement of the Case?" (As I said before, I think it makes sense that way, but I'd like to confirm.)

Reply:
Yes yes. AFTER the Supplemental Statement of the Case, it should say new Statement of the Case.

Reply:
Thanks @cvalarida @gnakm, updated.